db/migrate_core_cards/20141208132416_partial_reference_type.rb

```diff
@@ -0,0 +1,9 @@
+# -*- encoding : utf-8 -*-
+
+class PartialReferenceType < Wagn::CoreMigration
+  def up
+    Card.all.each do |card|
+      card.update_references
```

Comment:
fwiw, making this work in batches is currently the only thing I'm aware of that we need to get done before we merge this.  (though it would be good to get the tests passing; I'm going to work on that separately, since they're broken everywhere)